studio/src/ost.js

```diff
@@ -138,6 +138,7 @@ export function getOffferSelectorTool() {
                 <div id="ost"></div>
             </sp-dialog-wrapper>
         </overlay-trigger>
```

Comment:
is `</overlay-trigger>` closing tag needed? can't see the opening one

Reply:
nope ,thx

Reply:
@3ch023  thanks to you I found dead ost code to cleanup.